src/librustdoc/doctest.rs

```diff
@@ -933,7 +933,7 @@ impl Collector {
         if !item_path.is_empty() {
             item_path.push(' ');
         }
-        format!("{} - {}(line {})", filename.prefer_local(), item_path, line)
```

Comment:
There was some discussion that mixing few types of format parameters is harder to read: it need to track different places over code, instead sticking to one method. Or not?